runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -630,76 +630,86 @@ function! s:OpenHoverPreview(lines, filetype) abort
 
     let use_float_win = s:ShouldUseFloatWindow()
     if use_float_win
-        let bufname = nvim_create_buf(v:false, v:true)
-        call nvim_buf_set_lines(buf, 0, -1, v:true, lines)
-        let pos = getpos('.')
-
-        " Calculate width and height and give margin to lines
-        let width = 0
-        for index in range(len(lines))
-            let line = lines[index]
-            if line !=# ''
-                " Give a left margin
-                let line = ' ' . line
-            endif
-            let lw = strdisplaywidth(line)
-            if lw > width
-                let width = lw
-            endif
-            let lines[index] = line
-        endfor
-
-        " Give margin
-        let width += 1
-        let lines = [''] + lines + ['']
-        let height = len(lines)
-
-        " Calculate anchor
-        " Prefer North, but if there is no space, fallback into South
-        let bottom_line = line('w0') + winheight(0) - 1
-        if pos[1] + height <= bottom_line
-            let vert = 'N'
-            let row = 1
-        else
-            let vert = 'S'
-            let row = 0
+      let pos = getpos('.')
+
+      let margin = get(g:, "termdebug_floatMargin", 1)
```

Comment:
So I tested with 0 margin and 1 character wide margin, and for me, I think that adding a margin definitely add to the lisibility. Let me know what you think...
1 character wide margin:
![Screenshot_20190516_195351](https://user-images.githubusercontent.com/2647934/57875915-56a76200-7814-11e9-86ed-3a4c3fe4664e.png)
0 margin:
![Screenshot_20190516_195529](https://user-images.githubusercontent.com/2647934/57876001-92422c00-7814-11e9-8376-20dbb997d32e.png)

Reply:
With builtin enforced margin a lot more text is obscured by default. Completion popupmenu works without vertical margin lines. What I meant before is that the PR for 'margin' option will allow the user to configure this for _any_ float without explicit padding lines support added by every plugin that uses floats. (we should do the size allocation the correct way, so the plugin specifies the "inner" size of the window).

Reply:
ah okay, I understand now.

Let me revert that change.

Reply:
btw, in most terminals/fonts there should be some background space above the top of the f:s and ":s. You will have the same readability issues with popupmenu or statuslines with contrasting colors.

Reply:
arg... but I love this font :cry: